.github/workflows/repository-visualizer.yml

```diff
@@ -0,0 +1,20 @@
+name: Create repository visualisation diagram
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  get_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Update diagram
+        uses: githubocto/repo-visualizer@0.7.1
+        with:
+          excluded_paths: ".github, .netlify, .storybook, npm-shrinkwap.json,"
+          output_file: "public/diagram.svg"

```

Comment:
Ideally, this will go in the release.yml instead. @0-vortex will comment with more context.

Reply:
Hey there, sorry for the late reply, hit a few bumps in the road 😓 

You can safely delete this file now and add this to release.yaml:

```yaml
- name: "📊 repository visualizer"
  id: diagram
  uses: githubocto/repo-visualizer@0.7.1
  with:
    excluded_paths: "node_modules,build,.storybook,.netlify,.github,npm-shrinkwap.json"
    output_file: "public/diagram.svg"
    should_push: false
    artifact_name: "diagram"
```

right before line 59: https://github.com/open-sauced/open-sauced/blob/54ccd2a83d06137f7d76208f0985c594e69f7738/.github/workflows/release.yml#L59

Push this and the updated markdown link to public/diagram and we will take care of all the other steps